src/content/getting-started/cloud-provider-accounts/vmware-vsphere/_index.md

```diff
@@ -0,0 +1,84 @@
+---
+linkTitle: Preparing VMware vSphere for CAPI
+title: Preparing VMware vSphere for Cluster API
+description: How to set up your VMware vSphere Cluster tenant to run Giant Swarm management clusters and workload clusters under your jurisdiction.
+weight: 80
```

Comment:
This is the same `weight` as for the VCD article. Please change so they're explicitly alphabetically sorted.

Reply:
https://github.com/giantswarm/docs/pull/1927/commits/32b8dd1c77ee0fc48d9275e3a7cb04b188133d14